exporter/opentelemetry-exporter-otlp-proto-http/src/opentelemetry/exporter/otlp/proto/http/_log_exporter/__init__.py

```diff
@@ -146,7 +146,7 @@ def export(self, batch: Sequence[LogData]) -> LogExportResult:
 
             resp = self._export(serialized_data)
             # pylint: disable=no-else-return
-            if resp.status_code in (200, 202):
+            if resp.ok:
```

Comment:
From the [documentation](https://requests.readthedocs.io/en/latest/api/#requests.Response.ok) of `Response`, `resp.ok` also includes `3XX` return codes. Is this something that we actually want to handle as successful?

Reply:
It would depend on how the requests are made. If the `requests` follow the redirect, then `301`s will not appear here. At the minimum, we should accept all 2XX.

Reply:
Looking at the [code](https://github.com/gshiva/opentelemetry-python/blob/9ba12070e72ef0aef5efacb18b239cb12a3c1a3b/exporter/opentelemetry-exporter-otlp-proto-http/src/opentelemetry/exporter/otlp/proto/http/_log_exporter/__init__.py#L116), since we are not setting `allow_redirects=False`, `requests` by default will follow the redirect.

https://stackoverflow.com/questions/22150023/http-redirection-code-3xx-in-python-requests